digitalpalireader/content/js/dpr_pal.js

```diff
@@ -99,13 +101,18 @@ console.log('Loading DPR_PAL...');
     }
   }
 
+  DPR_PAL.setPaliTextContentHeight = () => {
+    bottomFrameUp?$("#paliTextContent").addClass("COLLAPSE"):$("#paliTextContent").removeClass("COLLAPSE");
```

Comment:
Some spaces between the ? and the : would make it more readable.

Reply:
Changed